src/it/resources/com/google/checkstyle/test/chapter4formatting/rule4854fieldannotations/InputFieldAnnotations.java

```diff
@@ -4,66 +4,51 @@
 import java.util.List;
 
 public class InputFieldAnnotations {
-    public @interface SomeAnnotation1 { }
+  public @interface SomeAnnotation1 {}
 
-    public @interface SomeAnnotation2 { }
+  public @interface SomeAnnotation2 {}
 
-    public @interface SomeAnnotation3 {
-        int x();
-    }
+  public @interface SomeAnnotation3 {
+    int x();
+  }
 
-    /**
-     * testing.
-     */
-    @SomeAnnotation1
-    @SomeAnnotation2
-    String name = "Zops";
+  /** testing. */
+  @SomeAnnotation1 @SomeAnnotation2 String name = "Zops";
 
-    /**
-     * testing.
-     */
-    @SomeAnnotation1 @SomeAnnotation2
-    int age = 19;
+  /** testing. */
+  @SomeAnnotation1 @SomeAnnotation2 int age = 19;
 
-    /**
-     * testing.
-     */
-    @SomeAnnotation1
-    @SomeAnnotation2 String favLanguage = "Java";
+  /** testing. */
+  @SomeAnnotation1 @SomeAnnotation2 String favLanguage = "Java";
 
-    /**
-     * testing.
-     */
-    @SomeAnnotation1
-    @SomeAnnotation3(x = 90) String favPet = "Dog";
+  /** testing. */
+  @SomeAnnotation1
+  @SomeAnnotation3(x = 90)
+  String favPet = "Dog";
 
-    @SomeAnnotation1 @SomeAnnotation2 boolean test = false;
+  @SomeAnnotation1 @SomeAnnotation2 boolean test = false;
 
-    @SuppressWarnings("all") @SomeAnnotation3(x = 0) float pi = 3.14f;
+  @SuppressWarnings("all")
+  @SomeAnnotation3(x = 0)
+  float pi = 3.14f;
 
-    @SomeAnnotation1 @SomeAnnotation3(x = 14)
-    /** // violation 'Javadoc comment is placed in the wrong location.'
-     * testing.
-     */
-    List<String> list = new ArrayList<>();
+  @SomeAnnotation1 @SomeAnnotation3(x = 14)
+  /** Testing. */ // violation 'Javadoc comment is placed in the wrong location.'
+  List<String> list = new ArrayList<>();
```

Comment:
Formatter modified this code to something like:

```java
  @SomeAnnotation1
  @SomeAnnotation3(x = 14)
  /** // violation 'Javadoc comment is placed in the wrong location.' testing. */
  List<String> list = new ArrayList<>();
```

but this way we were not able to detect the violation message so I modified it a little bit